src/pages/seat-reservation/data.js

```diff
@@ -0,0 +1,96 @@
+const cinemaHalls = [
+        {
+            numberOfSeats: { rows: 9, columns: 14 },
+            filmShowing: [
+                {
+                  id: 1,
+                  title: "Avengers",
```

Comment:
Jak rozumiem to tylko test. Docelowo trzeba zmienić id filmów - Judy przygotowała na Realtime Database w Firebase filmy z bazy omdb - mają id zaciągnięte z tej bazy i nimi mamy się posługiwać w projekcie.